README.rst

```diff
@@ -254,3 +254,55 @@ FAQ
 
 This can be done using "Attach to Local Process", but first you have to change the Godot binary filename to include :code:`python`, for example :code:`Godot_v3.0.2-stable_win64.exe` to :code:`python_Godot_v3.0.2-stable_win64.exe`.
 For more detailed guide and explanation see this `external blog post <https://medium.com/@prokopst/debugging-godot-python-with-pycharm-b5f9dd2cf769>`_.
+
+**How can I autoload a python script without attaching it to a Node?**
+
+In your :code:`project.godot` file, add the following section::
+
+  [autoload]
+  autoloadpy="*res://autoload.py"
```

Comment:
is the star `*` in the string normal ?

Reply:
Seems so:
https://github.com/godotengine/godot-demo-projects/blob/f61c760e629c81f570ec9be2eedd3f36c902ca12/visual_script/multitouch_view/project.godot#L19